syntax = "proto3";

package fbsrankings.messages.query;

import "fbsrankings/messages/options/options.proto";

message SeasonByYearValue {
    string season_id = 1;
    int32 year = 2;
}

message SeasonByYearResult {
    string query_id = 1;
    optional SeasonByYearValue season = 2;
}

message SeasonByYearQuery {
    string query_id = 1;
    int32 year = 2;

    option (fbsrankings.messages.options.topic) = "fbsrankings.query.season_by_year";
}
